### PR TITLE
correct spelling error

### PR DESCRIPTION
**Pull request by nuel-d1**

### CODE REVIEW

**Comment by nuel-d1** [comment]
> @googlebot I signed it!

**Comment by RFMVasconcelos** [comment]
Hey @nuel-d1!

Thank you for spotting this!
/ok-to-test
/lgtm

File: content/en/docs/aws/deploy/install-kubeflow.md [hunk 1]
```diff
@@ -177,7 +177,7 @@ This deployment may take 3-5 minutes to become ready. Verify that the address wo
 - **Dex**
   If you're using basic authentication, the credentials are the ones you specified in the KfDef file, or the default (`admin@kubeflow.org`:`12341234`). It is highly recommended to change the default credentials. To add static users or change the existing one, [add static users for basic auth](/docs/aws/deploy/install-kubeflow/#add-static-users-for-basic-auth).
 
-- **Coginito**
+- **Cognito**
```
Comment (PatrickXYS) [inline content/en/docs/aws/deploy/install-kubeflow.md:180]: Good catch!

**Comment by PatrickXYS** [comment]
@Jeffwan  Can you approve this?

**Pull request merged**